### PR TITLE
chore(doc): add more doc on thumbnail loading and error state

### PR DESCRIPTION
**Pull request by gcornut**
# General summary

Add more doc on thumbnail loading and error state

2022-03-02-180901_549x773_scrot

### CODE REVIEW

File: packages/site-demo/content/product/components/thumbnail/index.mdx [hunk 1]
```diff
@@ -34,9 +34,26 @@ Thumbnails come in two variants: `squared` and `rounded`
 
 <DemoBlock orientation="horizontal" demo="combined" />
 
-## Loading and error states
+## Loading skeleton
```
Comment (FlorentDinet) [inline packages/site-demo/content/product/components/thumbnail/index.mdx:37]: "Loading ~skeleton~state"

Skeleton is the component that make possible l[oading states](https://design.lumapps.com/product/patterns/loading-state/)

File: packages/site-demo/content/product/components/thumbnail/index.mdx [hunk 2]
```diff
@@ -34,9 +34,26 @@ Thumbnails come in two variants: `squared` and `rounded`
...
 
-<DemoBlock orientation="horizontal" demo="loading-error" withThemeSwitcher />
+<DemoBlock orientation="horizontal" demo="loading" withThemeSwitcher />
+
+While loading the image, the thumbnail displays a loading skeleton but only if the thumbnail has sufficient
```
Comment (FlorentDinet) [inline packages/site-demo/content/product/components/thumbnail/index.mdx:41]: We always use the plural for components.
And never write "you"

I'd re-write this part like that :

Thumbnails display a loading state while loading image. To display loading state, thumbnails require defined size.

For example:

File: packages/site-demo/content/product/components/thumbnail/index.mdx [hunk 3]
```diff
@@ -34,9 +34,26 @@ Thumbnails come in two variants: `squared` and `rounded`
...
+configuration to know what size to take. To do that, you can do any of the following:
+
+- use a fixed ratio (any ratio except `free` and `original`) and a width constraint (using `size` or manual CSS).
+- use `imgProps={{ width, height }}` where `width` and `height` is the original image size (or same original size ratio).
+  the thumbnail will scale down if the given `width` and `height` are bigger than allowed space
```
Comment (FlorentDinet) [inline packages/site-demo/content/product/components/thumbnail/index.mdx:46]: "Thumbnails will scale down..."

File: packages/site-demo/content/product/components/thumbnail/index.mdx [hunk 4]
```diff
@@ -34,9 +34,26 @@ Thumbnails come in two variants: `squared` and `rounded`
...
+
+## Error state
+
+If the image loading fails, the thumbnail will display an error state by default.
+
+<DemoBlock orientation="horizontal" demo="error" withThemeSwitcher />
+
+You can customize this state using the `fallback` prop to change the icon or even replace with a custom React node.
```
Comment (FlorentDinet) [inline packages/site-demo/content/product/components/thumbnail/index.mdx:54]: Error state can be customize using the `fallback` prop to change the icon or even replace with a custom React node.

**Review by FlorentDinet — CHANGES_REQUESTED** [review]
I've made suggestion to doc text.

We had [guidelines](https://www.notion.so/e1212cfc50ae4ae38f5a5fea3946d0bb) for writing doc but not accessible anymore, I'm trying to recover them

**Review by FlorentDinet — APPROVED** [review]
All good 👏

**Review by juanigalan91 — APPROVED** [review]

**Pull request merged**